pkg/cmd/build/new.go

```diff
@@ -54,6 +74,7 @@ func NewCmdBuildNew(f *factory.Factory) *cobra.Command {
 	cmd.Flags().StringVarP(&pipeline, "pipeline", "p", "", "The pipeline to build. This can be a {pipeline slug} or in the format {org slug}/{pipeline slug}.\n"+
 		"If omitted, it will be resolved using the current directory.",
 	)
+	cmd.Flags().BoolVarP(&confirm, "confirm", "", false, "Skip the confirmation step. Useful if being used in automation/CI")
```

Comment:
I'm wondering about the terminology of the "confirm" flag. It could be interpreted as "ask for confirmation". vs maybe `--no-confirm` meaning dont ask and just do it.

Do you have other ideas we could use? I don't really like `--yes` as it doesn't have much meaning either. tf uses `--auto-approve` which I dont like that much either, but its clearer